packages/fxa-shared/db/models/auth/security-event.ts

```diff
@@ -32,6 +32,8 @@ const EVENT_NAMES = {
   'account.secondary_email_added': 21,
   'account.secondary_email_removed': 22,
   'account.primary_secondary_swapped': 23,
+  'account.password_reset_otp_sent': 24,
+  'account.password_reset_otp_verified': 25,
```

Comment:
This is the fix.  I've added the migrations in earlier patches but I did not realize I need to hard code something here.

Reply:
I would probably leave a note at the top for future us.